docker/kernels/local/kernel.json

```diff
@@ -10,8 +10,9 @@
       "LD_LIBRARY_PATH": "/home/hadoop/local/gdal/lib",
       "PYSPARK_PYTHON": "/usr/bin/python3.4",
       "SPARK_HOME": "/usr/local/spark-2.1.0-bin-hadoop2.7",
+      "GEOPYSPARK_JARS_PATH": "/opt/jars",
       "PYTHONPATH": "/usr/local/spark-2.1.0-bin-hadoop2.7/python/lib/pyspark.zip:/usr/local/spark-2.1.0-bin-hadoop2.7/python/lib/py4j-0.10.4-src.zip",
-      "PYSPARK_SUBMIT_ARGS": "--master local[*] --archives /blobs/gdal-and-friends.tar.gz,/blobs/geopyspark-and-friends.tar.gz --jars /blobs/geotrellis-backend-assembly-0.1.0.jar --conf spark.serializer=org.apache.spark.serializer.KryoSerializer --conf spark.kyro.registrator=geotrellis.spark.io.kyro.KryoRegistrator --conf spark.executorEnv.LD_LIBRARY_PATH=/home/hadoop/local/gdal/lib/ pyspark-shell"
+      "PYSPARK_SUBMIT_ARGS": "--master local[*] --conf spark.executorEnv.LD_LIBRARY_PATH=/home/hadoop/local/gdal/lib/ pyspark-shell"
```

Comment:
Actually is the whole `PYSPARK_SUBMIT_ARGS` needed? `LD_LIBRARY_PATH` is given above as env var and I think `pyspark-shell` is assumed.